contracts/Core/CollectionManager.sol

```diff
@@ -176,6 +173,22 @@ contract CollectionManager is Initializable, CollectionStorage, StateManager, Co
         return collections[id];
     }
 
+    function getResult(bytes32 _name) external view override returns (uint32, int8) {
+        uint16 index = idToIndexRegistry[ids[_name]];
```

Comment:
hey I think L178 to L181 and L186 to L189 is repeated 
I think what can call getResultFromID only in getResult right ?

Reply:
hmm, might be possible, but in that case I would need to make getResultFromID public so we could use it here. But does it make change in gas cost consumption. what we can do make one internal function that gives result from ID and use that

Reply:
hey diff between external and public on gas costs is not applicable anymore I think 
From 0.8.0
Had seen some tweets on same 
Check this
https://twitter.com/transmissions11/status/1474540805922574337?s=20&t=IiO-3kknBzYfJGn42IvDaw

Reply:
So I think we can make it public without worrying about so

Reply:
sounds good